extension/lib/doc.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+(function(){
+var namespace = "lib/doc.js";
+if (window[namespace] === true) {
+    return;
+} else {
+    window[namespace] = true;
+}
+ 
+/*
+Class dedicated to keeping track of model of document. 
+Note: Called "Doc" because Document is already used in js. 
+*/
+
+class Doc {
+    constructor(readableDomIds) {
+        this.readableDomIds = readableDomIds; // string[]; List of all container IDs
+        this.containerId = null; // str; ID of current container
+        this.dict = null; // { str : int } , Set in calcTotalWords function
+        this.total_words = this.calcTotalWords(readableDomIds); // int; Total number of words in document;
+        this.keywords = this.setKeyWords(this.dict); // string[] keywords of document
+    };
+
+    // Returns: Total words in doc (int)
+    getTotalWords() {
+        return this.total_words;
+    }
+
+    // Returns: Container given container ID (jQuery element)
+    getContainer(containerId) {
```

Comment:
can you leave TODO: refactor?
i think this is redundant w/ tracker.js
i might create a domWrapper class that will be shared by both tracker and this guy, but not sure. would need to look at the final codebase after all the pulls

Reply:
Oh, please add the comment that this is redundant w/ tracker.js ' getContainer so I'll remember what the todo is about.